content/20.treatments.md

```diff
@@ -1,2 +1,49 @@
 ## Therapeutics {.page_break_before} 
-This section will include a review of information available about therapeutics.
+Given the rapid predicted spread of the disease, the development of therapeutics will be critical to mitigating its effect on health and the mortality rate.
+Typically, therapeutics can take a few forms.
+First, the treatment and reduction of symptoms can result in the reduction of the severity and risk associated with an active infection.
+Second, the development of antiviral drugs can drive a reduced recovery time for patients by inhibiting the development of the virus once an individual is infected.
+Finally, vaccines present a strategy for bolstering the immune response of the populus broadly to the virus, resulting in a lower rate of infection.
+All three of these strategies have been valuable elements of responses to other viruses, including coronaviruses, and are being investigated by researchers at present.
+
+### Treatment of Symptoms
+COVID-19 is characterized by...
+The most severe and concerning symptoms are typically...
+The symptoms most often regarded as the proximal cause of death from COVID-19 are...
+Other diseases with similar symptoms include XYZ but these diseases may be different because...
+
+So far, some strategies for reducing the severity of symptoms have included...
+
+#### Relevant Mechanisms and Possible Pharmaceutical Interventions
+[Given what we know about the mechanisms of the virus and why it produces the symptoms we see, are there drugs or categories of intervention that might be relevant?]
+
+#### Data-Driven Prediction of Interventions
+[The machine learning community has been called upon to try to predict possible interventions. What are they doing and what are they finding about treating symptoms?]
+
+### Antivirals
+Antivirals are an emerging category of drugs.
+Unlike antibiotics, they do not kill viruses.
+Rather, they inhibit the proliferation of a virus.
+Antivirals have been used to treat XYZ diseases through [what mechanisms or approaches are used?]
+
+#### Antivirals and Coronoviruses
+[Have antivirals been used to treat other coronaviruses? What went well/badly? How do these interventions work?]
+
+#### Antiviral Targeting of SARS-CoV-2
```

Comment:
I'd probably make these types of synthesis questions all part of the section without any separation and then have the subsections be the names of potential therapies plus the evidence around them.

Reply:
May have gone overboard but it's now one section per topic (symptoms, antivirals,  vaccines) and a direction to make new categories as appropriate